tests/test_borrowed_get.py

```diff
@@ -51,23 +51,70 @@ def test_borrowed_get_all_with_user_token(test_db: sessionmaker) -> None:
     register_user(TEST_USER)
     token = login(TEST_USER_AUTH).json()["token"]
 
-    response = make_request(token)
+    response = make_request("/borrowed/", token)
     assert response.status_code == status.HTTP_401_UNAUTHORIZED
 
 
+def test_borrowed_get_by_id(test_db: sessionmaker) -> None:
+    """
+    Tests the get borrowed by id endpoint
+    """
+
+    borrowed = create_borrowed(test_db)
+    token = login(TEST_USER_AUTH).json()["token"]
+
+    response = make_request(f"/borrowed/{borrowed.id}", token)
+    data = response.json()
+    assert response.status_code == status.HTTP_200_OK
+    assert "copy_id" in data and data["copy_id"] == borrowed.copy_id
+    assert "user_id" in data and data["user_id"] == borrowed.user_id
+    assert "due_date" in data and data["due_date"] == borrowed.due_date.isoformat()
+    assert (
+        "issue_date" in data and data["issue_date"] == borrowed.issue_date.isoformat()
+    )
+
+
+def test_borrowed_get_by_id_without_token(test_db: sessionmaker) -> None:
+    """
+    Tests the get borrowed by id endpoint without a token. Should return 401
+    """
+    borrowed = create_borrowed(test_db)
+
+    response = make_request(f"/borrowed/{borrowed.id}")
+    response.status_code == 401
```

Comment:
Use status codes from Starlette , i guess you missed it here !! This is fine aswell .

Reply:
Fixed it. Thanks for pointing out.